module_utils/ansible_nailgun_cement.py

```diff
@@ -201,6 +251,16 @@ def find_organization(module, name, failsafe=False):
     return handle_find_response(module, response, message="No organization found for %s" % name, failsafe=failsafe)
 
 
+def find_location(module, name, failsafe=False):
+    loc = Location(name=name).search()
+    return handle_find_response(module, loc, message="No location found for %s" % name, failsafe=failsafe)
```

Comment:
This method does not work as implemented. Location and Organization are both of the Taxonomy class, and (for whatever reason), their search APIs are slightly different than other models. This method should be something like this:

```py
def find_location(module, name, failsafe=False):
    response = Location().search(set(), {'search': 'name="{}"'.format(name)})
    return handle_find_response(module, response, message="No location found for %s" % name, failsafe=failsafe)
```